test/production/app-dir/ppr-disable-streaming-metadata/ppr-disable-streaming-metadata.test.ts

```diff
@@ -0,0 +1,38 @@
+import { nextTestSetup } from 'e2e-utils'
+
+describe('ppr-disable-streaming-metadata', () => {
```

Comment:
I'd probably just remove this test since we aren't planning on it being like this, it's just temporary.